<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureProjectBuildActionsDialog</class>
 <widget class="QDialog" name="ConfigureProjectBuildActionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Project Build Actions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="addBuildActionButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="asproject.qrc">
         <normaloff>:/images/pba_add.png</normaloff>:/images/pba_add.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editBuildActionButton">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="asproject.qrc">
         <normaloff>:/images/pba_edit.png</normaloff>:/images/pba_edit.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteBuildActionButton">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="asproject.qrc">
         <normaloff>:/images/pba_delete.png</normaloff>:/images/pba_delete.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="envPushButton">
       <property name="text">
        <string>Environment</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="buidActionsTableView"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>addBuildActionButton</tabstop>
  <tabstop>editBuildActionButton</tabstop>
  <tabstop>deleteBuildActionButton</tabstop>
  <tabstop>buidActionsTableView</tabstop>
 </tabstops>
 <resources>
  <include location="asproject.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureProjectBuildActionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureProjectBuildActionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
